akka-persistence/src/main/scala/akka/persistence/state/DurableStateStoreRegistry.scala

```diff
@@ -85,9 +85,7 @@ class DurableStateStoreRegistry(system: ExtendedActorSystem)
    * Java API: Returns the [[akka.persistence.state.javadsl.DurableStateStore]] specified by the given
    * configuration entry.
    */
-  final def getDurableStateStoreFor[T <: javadsl.DurableStateStore[_]](
-      @unused clazz: Class[T], // FIXME generic Class could be problematic in Java
```

Comment:
I think it was related to that in Java you can't easily create/get a Class with a generic parameter. E.g. this would not work:
```
Class<DurableStateStoreQuery<Record>> durableStateStoreQueryClass = DurableStateStoreQuery.class;
```

We get away by using this in the doc sampel:

```
DurableStateStoreQuery<Record> durableStateStoreQuery =
        DurableStateStoreRegistry.get(system)
            .getDurableStateStoreFor(DurableStateStoreQuery.class, pluginId);
```

That is an untyped warning, but probably fine.